make/tests

```diff
@@ -133,6 +133,8 @@ src/test/unit/mcmc/hmc/hamiltonians/diag_e_metric_test.cpp: src/test/test-models
 src/test/unit/mcmc/hmc/hamiltonians/unit_e_metric_test.cpp: src/test/test-models/good/mcmc/hmc/hamiltonians/funnel.hpp
 src/test/unit/mcmc/hmc/integrators/expl_leapfrog_test.cpp: src/test/test-models/good/mcmc/hmc/integrators/command.hpp
 src/test/unit/mcmc/hmc/integrators/expl_leapfrog2_test.cpp: src/test/test-models/good/mcmc/hmc/integrators/gauss.hpp
+src/test/unit/mcmc/hmc/integrators/impl_leapfrog_test.cpp: src/test/test-models/good/mcmc/hmc/integrators/command.hpp
+src/test/unit/mcmc/hmc/integrators/impl_leapfrog2_test.cpp: src/test/test-models/good/mcmc/hmc/integrators/gauss.hpp
```

Comment:
Need to add a few more test dependencies:
- `src/test/unit/mcmc/hmc/hamiltonians/softabs_metric_test.cpp`
- `src/test/unit/mcmc/hmc/nuts/softabs_nuts_test.cpp`
- ` src/test/unit/mcmc/hmc/xhmc/softabs_xhmc_test.cpp`